.circleci/config.yml

```diff
@@ -99,6 +99,9 @@ jobs:
       - run: yarn build:production
       - run: yarn test-build
       - *persist_deploy_workspace
+    environment:
+      # See: https://github.com/mozilla/addons-blog/issues/351
+      AMO_BASE_URL: https://addons-dev.allizom.org
```

Comment:
We're going to have the same issue with `build-blog-stage`, right ?

Reply:
We only want to override the API used on dev. For stage we want to continue to use the prod API, because stage is important for testing what's going to end up on prod. For this reason, we do not need similar logic for `build-blog-stage`.